src/PKSim.Core/Services/ParameterTask.cs

```diff
@@ -483,7 +483,7 @@ public ICommand UpdateParameterValueOrigin(IParameter parameter, ValueOrigin val
 
       public ICommand SetParameterValueOrigin(IParameter parameter, ValueOrigin newValueOrigin)
       {
-         return setParameterValueOrigin(parameter, newValueOrigin, shouldChangeVersion: false);
+         return setParameterValueOrigin(parameter, newValueOrigin, shouldChangeVersion: true);
```

Comment:
I had implemented the action as NOT change version at the beginning and forgot to update the code once we changed our mind